dom.bs

```diff
@@ -6641,6 +6651,9 @@ string <var>namespace</var> (default null):</p>
 <a for=/>attribute</a> <var>attr</var> and an <a for=/>element</a> <var>element</var>:
 
 <ol>
+ <li><p>Let <var>verifiedValue</var> be the result of calling <a>verify attribute value</a>
+ <var>attr</var>'s <a for=Attr>value</a> for <var>attr</var>, with <var>element</var>.
```

Comment:
```suggestion
 <li><p>Let <var>verifiedValue</var> be the result of <a>verifying attribute value</a> given
 <var>attr</var>'s <a for=Attr>value</a>, <var>attr</var>, and <var>element</var>.
```
However, this doesn't match the signature above. What gives?

Reply:
I've updated this and the signature to hopefully be clearer. I think it does match the signature it was just written rather poorly. I've changed the signature to a more convetional one and updated the call sites accordingly.